### DISCUSSION TITLE
LOOKING FOR PROJECTS

### DISCUSSION THREAD

**Discussion by borjagamboa** [community/community#125373] [New to GitHub]
### Hi!

I haven't coded for some years and now that I am unemployed and trying to find something I really feel like, guess what showed up. I actually used to enjoy coding in Python, but I don't exactly why I decided to pursue a MBA. I reckon I succumbed to the image we are sold of a businessman.

Either way, I have been doing some courses on Algorithmic Trading as a way to recall the foundations of Python and I would like to develop further in this area.

## Does anybody know about a project or a way to contribute to the community and learn at the same time?

Looking forward to your replies

**Comment by judahpaul16** [comment]
Check out my project GPT Home 🤖💬: https://github.com/judahpaul16/gpt-home
